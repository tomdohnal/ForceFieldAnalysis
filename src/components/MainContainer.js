import React from 'react';
import { Grid } from 'semantic-ui-react';

import DrivingForcesContainer from './DrivingForcesContainer';
import HinderingForcesContainer from './HinderingForcesContainer';
import ChangeContainer from '../containers/ChangeContainer';
import { HEADER_HEIGHT } from '../constants/styles';

const MainContainer = () => (
  <Grid style={{ height: `calc(100% - ${HEADER_HEIGHT}` }}>
    <Grid.Row>
      <Grid.Column width={6}>
        <DrivingForcesContainer />
      </Grid.Column>
      <Grid.Column width={4}>
        <ChangeContainer />
      </Grid.Column>
      <Grid.Column width={6}>
        <HinderingForcesContainer />
      </Grid.Column>
    </Grid.Row>
  </Grid>
);

export default MainContainer;
