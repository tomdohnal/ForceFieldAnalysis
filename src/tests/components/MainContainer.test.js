import React from 'react';
import { shallow } from 'enzyme';

import MainContainer from '../../components/MainContainer';

describe('MainContainer', () => {
  it('renders correctly', () => {
    const tree = shallow(<MainContainer />);

    expect(tree).toMatchSnapshot();
  });
});
