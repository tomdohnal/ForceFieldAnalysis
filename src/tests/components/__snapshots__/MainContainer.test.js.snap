// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer renders correctly 1`] = `
<Grid
  style={
    Object {
      "margin": 0,
      "minHeight": "calc(100% - 60px",
    }
  }
>
  <GridRow
    style={
      Object {
        "padding": "8px",
      }
    }
  >
    <GridColumn
      width={6}
    >
      <DrivingForcesContainer />
    </GridColumn>
    <GridColumn
      width={4}
    >
      <Connect(ChangeContainer) />
    </GridColumn>
    <GridColumn
      width={6}
    >
      <HinderingForcesContainer />
    </GridColumn>
  </GridRow>
</Grid>
`;
